app/src/main/java/io/gnosis/safe/ui/StartActivity.kt

```diff
@@ -66,9 +67,10 @@ class StartActivity : BaseActivity(), SafeOverviewNavigationHandler {
                         safeRepository.setActiveSafe(it)
                         setSafeData(it)
                     }
-                    //TODO: figure out which tab (QUEUE vs HISTORY) to open
                     if (txId == null) {
-                        Navigation.findNavController(this@StartActivity, R.id.nav_host).navigate(R.id.transactionsFragment)
+                        Navigation.findNavController(this@StartActivity, R.id.nav_host).navigate(R.id.transactionsFragment, Bundle().apply {
+                            putInt("activeTab", TxPagerAdapter.Tabs.HISTORY.ordinal) // open history tab
+                        })
```

Comment:
For assets and settings we do not want this ATM?

Reply:
what do you mean? We have those enums also for assets and settings